dashboard/app/models/ability.rb

```diff
@@ -96,6 +96,7 @@ def initialize(user)
       can :create, UserLevel, user_id: user.id
       can :update, UserLevel, user_id: user.id
       can :create, StudentWorkEvaluation, user_id: user.id
+      can :create, StudentWorkEvaluationSummary
```

Comment:
I take it "anyone" here can create a `StudentWorkEvaluationSummary` because it is automatically created when a `UserLevelEvaulation` is generated (which is required to be generated by someone with `teacher` permissions.  So... it is not the user really generating the `StudentWorkEvaluationsSummary` as much as an algorithm generating a `StudentWorkEvaluationsSummary` when a `UserLevelEvaulation` is generated by the teacher.

^^ This is just me checking my own understanding.  I hadn't worked with `ability.rb` before much so it took me a while to understand what this line was doing and why it was needed.

Reply:
A teacher can create a `StudentWorkEvaluation` on behalf of their student (like we're doing for free responses when they click the "Evaluate student responses button") or a student can create a `StudentWorkEvaluation` for their own work (which we do "behind the scenes" for a couple of coding levels when they click "run"). 
 
 Yes, because the `logStudentWorkEvaluation` function combines creating the `StudentWorkEvaluations` and creating any summaries that are needed the user is implicitly creating the summaries when they hit that endpoint.